lib/spack/docs/containers.rst

```diff
@@ -179,7 +190,211 @@ of environments:
          app: "gromacs"
          mpi: "mpich"
 
-The tables below describe the configuration options that are currently supported:
+A detailed description of the options available can be found in the
+:ref:`container_config_options` section.
+
+-------------------
+Setting Base Images
+-------------------
+
+The ``images`` subsection is used to select both the image where
+Spack builds the software and the image where the built software
+is installed. This attribute can be set in two different ways and
+which one to use depends on the use case at hand.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Use Official Spack Images From Dockerhub
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To generate a recipe that use an official Docker image from the
+Spack organization to build the software and the corresponding official OS image
+to install the built software, the user is just requested to
+specify:
```

Comment:
Suggested wording change: "the user is just requested to specify" -> "all the user has to do is specify"